index.js

```diff
@@ -0,0 +1,77 @@
+import luxon from './modules/luxon.js';
+
+import AwesomeBooks from './modules/book.js';
+
+const awesomeBooks = new AwesomeBooks();
+awesomeBooks.load();
+if (awesomeBooks.getBooks().length < 1) {
+  awesomeBooks.addBook('The Power of Focus', 'Mark Victor Hansen');
+}
+
+const bookListSection = document.querySelector('#book-list');
+
+function renderBookList() {
```

Comment:
- [ ] Please use ES6 arrow function syntax here instead of the `function` keyword.